### ISSUE TITLE
`goto` giving wrong path for modules that are `npm link`ed

### ISSUE THREAD

**Issue by trotzig** [Galooshi/import-js#279]
If I use linked dependencies in my project, `goto` will give back an absolute path leading to the  `node_modules` folder of import-js. Here's me using an `npm link`ed dependency, `chokidar`:

```
> cd ~/chokidar
> npm link
> cd ~/my-project
> npm link chokidar
> importjs goto chokidar some-file.js
{ [...] "goto": "/Users/henrictrotzig/import-js/node_modules/chokidar/index.js" }
```

The resulting goto in the json output is what `require.resolve` tells us. I'm surprised to see import-js being part of the resulting (absolute) path. That doesn't happen for dependencies that aren't `npm link`ed (in which case I get a relative path, e.g. `node_modules/chokidar/index.js`).

Since importjs is also `npm link`ed for me locally (which might interact weirdly here), the path might be different for folks who have `npm install`ed import-js.

**Comment by ljharb** [comment]
I think `require.resolve` runs from the PWD, whereas you'll want it to run from the directory the file is in. You may need `process.chdir` before running `require.resolve`?

**Comment by trotzig** [comment]
This will be fixed when 1ce9408c37 is merged.

**Comment by trotzig** [comment]
Not sure why I reopened this again, as I believe it has been fixed with 2.1
